<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Braini</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">font-family: Noto Sans SC;
</string>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="roll_label">
        <property name="text">
         <string>Roll: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="roll_lineEdit">
        <property name="minimumSize">
         <size>
          <width>133</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="pitch_label">
        <property name="text">
         <string>Pitch: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pitch_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="yaw_label">
        <property name="text">
         <string>Yaw: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="yaw_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="submit_button">
      <property name="text">
       <string>Submit</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTableWidget" name="table_data">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="autoScrollMargin">
         <number>17</number>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Roll</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Pitch</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Yaw</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="lineChartWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="testicon.qrc"/>
 </resources>
 <connections/>
</ui>
